onnxruntime/core/providers/cpu/controlflow/if.cc

```diff
@@ -90,15 +91,56 @@ class IfImpl {
   std::vector<std::pair<AllocationType, MLValue>> outputs_;
 };
 
+void If::CheckForPassthroughOptimization(const OpKernelInfo& info,
+                                         const ONNX_NAMESPACE::GraphProto& then_proto,
+                                         const ONNX_NAMESPACE::GraphProto& else_proto) {
+  // check if subgraph has a single identity node passing a single input through as a single output
+  auto check_identity_passthrough = [](const ONNX_NAMESPACE::GraphProto& proto) -> const std::string {
+    if (proto.node_size() == 1 && proto.output_size() == 1) {
+      const auto& node = proto.node().Get(0);
+      if (node.op_type() == "Identity") {
```

Comment:
There should be a constant definition for "Identity" string (used in identity elimination rule). let's reuse it.

Reply:
It's not in a shareable place. I can define it somewhere more common if you want.

```
class EliminateIdentity : public RewriteRule {
...
 private:
  /** Apply rule when op type is one of the following. */
  const std::string included_op_type_ = "Identity";
```